recipe/meta.yaml

```diff
@@ -38,6 +38,25 @@ test:
     - test -f ${PREFIX}/lib/pkgconfig/tango.pc
     - test -f ${PREFIX}/include/tango/tango.h
 
+outputs:
+  - name: cpptango
+    files:
+      - lib/libtango.so
+      - lib/libtango.so.9
+      - lib/libtango.so.{{ version }}
+      - lib/pkgconfig/tango.pc
+      - include/tango
+
+  - name: cpptango-dbg
+    requirements:
+      run:
+        - {{ pin_subpackage('cpptango', exact=True) }}
+    files:
+      - lib/libtango.so.{{ version }}.dbg
```

Comment:
Would we have to change the command line arguments for the linker to use this, or is it something the loader does automatically?
I'm wondering if the PyTango build steps would have to change to use the debug info.  E.g., this line in [setup.py](https://gitlab.com/tango-controls/pytango/-/blob/44a3eb854637d68a336ee87c9988deb9c1b85be6/setup.py#L471).

Reply:
I think it should be automatic thanks to the `--add-gnu-debuglink` option that points to the debug symbols file.
If you have time to try, I uploaded the build result to my personal channel. You can install locally using:
`conda create -y -n tango -c conda-forge -c beenje cpptango-dbg`